src/main/java/io/pravega/connectors/hadoop/PravegaOutputFormat.java

```diff
@@ -91,12 +91,15 @@ private Object getInstanceFromName(String className) throws IOException {
     }
 
     @VisibleForTesting
-    protected EventStreamClientFactory getClientFactory(String scope, URI controllerUri) {
-        ClientConfig clientConfig = ClientConfig.builder().controllerURI(controllerUri).build();
+    protected EventStreamClientFactory getClientFactory(String scope, ClientConfig clientConfig) {
+        if (clientConfig == null) {
```

Comment:
Initializing the clientConfig should be a responsibility of the caller. Or, if we want to do it in this class, we should add an overloaded method that just takes scope as an input and calls this method after creating an empty client builder.

Reply:
I was using this to address an issue with the mock instance. I have changed the test code to fix the original issue and removed this line.